examples/travelapp-kogito-travel-agency.yaml

```diff
@@ -1,21 +1,25 @@
 apiVersion: app.kiegroup.org/v1alpha1
-kind: KogitoApp
+kind: KogitoBuild
 metadata:
   name: kogito-travel-agency
 spec:
-  enableEvents: true
-  enablePersistence: true
+  type: RemoteSource
+  gitSource:
+    uri: 'https://github.com/kiegroup/kogito-travel-agency-tutorial/'
```

Comment:
Do you mind updating to: https://github.com/kiegroup/kogito-examples/tree/master/kogito-travel-agency

Reply:
done